core/app/services/spree/export/google_rss.rb

```diff
@@ -0,0 +1,133 @@
+require 'nokogiri'
+
+module Spree
+  module Export
+    class GoogleRss
+      def call(options)
+        @store = Spree::Store.find(options.spree_store_id)
```

Comment:
I think we could add here some checking and return error when we don't find `Store` by `id`.